repository-data/webfiles/src/main/resources/site/freemarker/common/apicatalogue/api-catalogue.ftl

```diff
@@ -20,10 +20,21 @@
 <@headerBanner document />
 
 <div class="nhsd-t-grid nhsd-!t-margin-top-6 nhsd-api-catalogue">
-
     <div class="nhsd-t-row">
         <div class="nhsd-t-col-12">
             <#if document.body?has_content??>
+                <#if showAll??>
+                    <#if showAll == true>
+                        <#assign toShowAll="false">
+                    <#else>
+                        <#assign toShowAll="true">
+                    </#if>
+                </#if>
+
+                <@hst.renderURL var="showAlllink">
+                    <@hst.param name="showAll" value="${toShowAll}" />
```

Comment:
If you replace:
```
${toShawAll}
```
with:
```
${showAll?string('false', 'true')}
```
you'll be able to get rid of the whole `<# if showAll??> ... </#if>` construct and the `toShowAll` variable.